src/main/java/com/google/api/codegen/config/MethodConfig.java

```diff
@@ -333,6 +334,12 @@ public static boolean isGrpcStreamingMethod(Method method) {
     return method.getRequestStreaming() || method.getResponseStreaming();
   }
 
+  /** Returns true if the method returns empty message */
+  public static boolean isReturnEmptyMessageMethod(Method method) {
+    MessageType returnMessageType = method.getOutputMessage();
+    return returnMessageType.getFields().size() == 0;
```

Comment:
You don't have to check like this -- we should only special-handle the `google.protobuf.Empty` type.  For example, Java returns `AddLabelResponse`. See https://github.com/googleapis/toolkit/blob/master/src/test/java/com/google/api/codegen/testdata/java_main_library.baseline#L2118